content/finalist/ttjc@finalist@cmcodes1/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: cmcodes
+type: "finalist"
+name: "Chandra Mohan Pandey"
+portfolio: "https://cmcodes1.github.io"
+github-link: "https://github.com/cmcodes1"
+linkedin-link: "https://linkedin.com/in/cmcodes"
+looking-for: "job"
+description: "Analyst | Tech Geek | Developer"
+image-link: "https://avatars2.githubusercontent.com/u/34341671?s=460&u=73e18d14ad731e271dcb5a16e215cc7cf626721b&v=4"
+---
+
+A guy who's crazy about React. ⚛
+
+### introduction
+
+A professional in the field of coding since the date of birth, who has a great affinity towards making useful stuff (mostly with JS); fueled with lots and lots of optimism and self-motivation. 💪
+
+Wrote a piece of code 7 years ago, it was fun; so that’s what I have been doing till date. Also, a huge Marvel fan! 😁
+
+## why are you taking this challenge?
+
+Looking for a job.
+
+## something interesting about you?
+
+I am an INTJ.
+National Youth Parliament Speaker.
+I lead a community of 10k+ developers.
+Going hard on mobile app dev with React Native.
+
+### help I need?
+
+It would be good if someone recommends me some books on JS.
+
+### how to contact me?
+
+- [E-mail](mailto:chandramohan.pandey2015@gmail.com)
+
+- [Twitter](https://twitter.com/cmcodes1)
+
+- [Instagram](https://instagram.com/cmcodes)
+
+- [LinkedIn](https://linkedin.com/in/cmcodes)
+
+### projects
+
+My projects:
+
+#### Studify
+
+_description_ A student utility android application.
+
+_stack_ React Native
+
+_github link_ https://github.com/cmcodes1/Studify
```

Comment:
Any deployment link or for preview provide release link.

Reply:
It has already been pushed to Google Play Console, but will be available in 6 days, after they review it. So, I can't provide any link for that yet! Will update it when it gets approved from Play Store. Although, I do have attached the screenshots of the app in the README.md file of the mentioned GitHub link.

Reply:
provide any link to download the apk at present later update when it gets approved